app/src/main/java/com/sopt/smeem/presentation/mypage/mysummary/MySummaryScreen.kt

```diff
@@ -54,6 +57,7 @@ fun MySummaryScreen(
         if (isDataChanged) {
             viewModel.fetchMySummaryData()
             viewModel.setDataChanged(false)
+            eventVM.sendEvent(AmplitudeEventType.ACHIEVEMENT_VIEW)
         }
     }
```

Comment:
영역을 따로 만들어야 될 것 같습니다 isDataChanged가 true 되는 상황이 트레이닝 플랜을 변경했을때 뿐이라 scope를 분리해주는게 좋을 것 같아요~
```suggestion
        LaunchedEffect(true) {
            eventVM.sendEvent(AmplitudeEventType.ACHIEVEMENT_VIEW)
        }
    }
```

Reply:
반영했아요